build.xml

```diff
@@ -68,6 +68,23 @@
 		</jar>
 	</target>
 
+	<target name="slim-jar" depends="compile-slim" description="generate slim only jar file">
```

Comment:
Can you convert this to the build.gradle currently in use?

Reply:
👍